DemoApp/Scenes/Discover/DiscoverViewModel.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  DiscoverViewModel.swift
+//  DemoApp
+//
+//  Created by Gabi Matko on 09.02.2021.
+//
+
+import Foundation
+import Bond
+import ReactiveKit
+import Alamofire
+
+class DiscoverViewModel {
+    
+    // MARK: Types
+    
+    enum InfoCommand {
+        case reloadUI
+    }
+
+    typealias Command =  InfoCommand
+    
+    // MARK: Properties
+    
+    let repo: DrinksRepoProtocol
+    
+    // MARK: Observables
+    
+    let drinks = MutableObservableArray<DrinkCellViewModel>([])
+    
+    let command = Observable<DiscoverViewModel.Command?>(nil)
+    
+    // MARK: Init
+    
+    init(repo: DrinksRepoProtocol) {
+        self.repo = repo
+    }
+    
+    func loadData() {
+        repo.loadDrinks { drinks in
+            self.drinks.removeAll()
+            self.drinks.insert(contentsOf: drinks, at: 0)
```

Comment:
Wouldn't this be easier to do? 🤔 
```suggestion
            self?.drinks.replace(with: drinks)
```